Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/CoinsConfig.yaml

```diff
@@ -0,0 +1,48 @@
+imports:
+  DriverCoinsFunctionType: Dashboard.Common.DriverCoins
+  CoinsConfig: Domain.Types.Coins.CoinsConfig
+
+module: CoinsConfig
+
+apis:
+  - PUT: # Updating Entries
+      endpoint: updateEntries/{active}/{expirationAt}/{coins}
+      auth: ApiAuthV2
+      params:
```

Comment:
lets use request instead of too many params
especially since these params should be optional for `update` api

Reply:
id should be included also in req, because we should know which entry to update